### PR TITLE
Fix material date time picker ok cancel color

### PR DESCRIPTION
**Pull request by osodebailar**
Ok Cancel colors are now equal to the other dialogs .
Other colors can be discussed in issue https://github.com/nightscout/AndroidAPS/issues/1636

### CODE REVIEW

File: automation/src/main/java/info/nightscout/androidaps/plugins/general/automation/elements/InputDateTime.kt [hunk 1]
```diff
@@ -60,6 +61,7 @@ class InputDateTime(private val rh: ResourceHelper, private val dateUtil: DateUt
                                 val cal = Calendar.getInstance().apply { timeInMillis = value }
                                 val clockFormat = if (DateFormat.is24HourFormat(context)) TimeFormat.CLOCK_24H else TimeFormat.CLOCK_12H
                                 val timePicker = MaterialTimePicker.Builder()
+                                    .setTheme(info.nightscout.androidaps.core.R.style.MaterialTimePickerTheme)
```
Comment (MilosKozak) [inline automation/src/main/java/info/nightscout/androidaps/plugins/general/automation/elements/InputDateTime.kt:64]: shorten the string

Comment (osodebailar) [reply automation/src/main/java/info/nightscout/androidaps/plugins/general/automation/elements/InputDateTime.kt]: > shorten the string

Done

File: automation/src/main/java/info/nightscout/androidaps/plugins/general/automation/elements/InputDateTime.kt [hunk 2]
```diff
@@ -60,6 +61,7 @@ class InputDateTime(private val rh: ResourceHelper, private val dateUtil: DateUt
                                 val cal = Calendar.getInstance().apply { timeInMillis = value }
                                 val clockFormat = if (DateFormat.is24HourFormat(context)) TimeFormat.CLOCK_24H else TimeFormat.CLOCK_12H
                                 val timePicker = MaterialTimePicker.Builder()
+                                    .setTheme(info.nightscout.androidaps.core.R.style.TimePicker)
```
Comment (MilosKozak) [inline automation/src/main/java/info/nightscout/androidaps/plugins/general/automation/elements/InputDateTime.kt:64]: still here

Comment (osodebailar) [reply automation/src/main/java/info/nightscout/androidaps/plugins/general/automation/elements/InputDateTime.kt]: > still here

fixed

**Pull request merged**